Source/Transforms/LMCWindowFunctions.cc

```diff
@@ -0,0 +1,170 @@
+/*
+ * LMCWindowFunctions.cc
+ *
+ *  Created on: Jan 10, 2022
+ *      Author: A. B. Telles
+ */
+#include <iostream>
+#include <fstream>
+
+#include "logger.hh"
+#include "LMCWindowFunctions.hh"
+#include "LMCConst.hh"
+
+namespace locust
+{
+    LOGGER( lmclog, "WindowFunctions" );
+
+    WindowFunctions::WindowFunctions():
+    IsWindowGeneratedFlag( false ),
+    fTotalWindowSize(0),
+    fPreWindowBins(0),
+    fSignalSize(0),
+    fWindowFunctionType(0),
+    fWindowFunction( 1 ),
+    fTukeyWindowAlpha( 0.5 )
+    {
+    }
+    
+    WindowFunctions::~WindowFunctions()
+    {
+    }
+    
+    bool WindowFunctions::Configure(const scarab::param_node& aParam)
```

Comment:
I'm a little confused why this is here. It isn't configuring anything, so is this just leftovers from previous versions of the code?

Reply:
For right now it is a useful placeholder that gets run once after Locust is started.  If the WindowFunctions class needs parameterizable class variables in the future, they can easily be located here.